docs/specviz/displaying.rst

```diff
@@ -2,10 +2,18 @@
 Displaying Spectra
 ******************
 
-Words go here
-
-
-
+Spectra loaded by the user will be automatically displayed in the viewer, 
+although additional spectra loaded after the first may not be fully shown 
+if they exceed the bounds of the plotted area, which are set based on the 
+first displayed spectrum. The bounds can be changed via the 
+:ref:`Pan/Zoom <pan-zoom>` tool or by deselecting the current spectra and 
+selecting a different spectrum for display. Spectra generated by plugins 
+(e.g a spectrum generated by the Gaussian Smooth plugin) will generally be 
+automatically be displayed as well, and one can always see the spectra 
+available and toggle their visibility in the data selection dropdown menu 
+(see :ref:`Selecting Data Set<selecting-data-set>` for more detail).
+
+.. _selecting-data-set:
```

Comment:
Looks like not having a newline after this is causing errors. What is the purpose of this line?

Reply:
Adding a line break got the internal links working, thanks! I'll rebase in a moment

Reply:
And to clarify, the purpose is so that that section can be referenced in other parts of the documentation in a way that automatically generates a nice link to it.

Reply:
Got it, now I know how to do that, thanks!